### ISSUE TITLE
Training with EG1800 dataset using alpha mask

### ISSUE THREAD

**Issue by anilsathyan7** [xuebinqin/U-2-Net#189]
I trained the model with [EG1800](https://onedrive.live.com/?authkey=%21ADkS4V32BUmspOg&cid=F5111408123B1D9C&id=F5111408123B1D9C%2115035&parId=F5111408123B1D9C%2115033&action=locate) dataset which contains ground truth mask with values [0.0....1.0] i.e alpha.  It contains 1500 train and 300 validation images. After 100 epochs the train loss was around 0.2; but val loss was around 35. I used the provided weights to initialize the training process. It seems to overfit to the training set. I used a similar code for validation also.  Is it required that the mask should be strictly binary?
